bgpd/bgp_vty.c

```diff
@@ -7508,16 +7511,25 @@ DEFUN (show_bgp_vrfs,
 			json_object_string_add(
 				json_vrf, "rmac",
 				prefix_mac2str(&bgp->rmac, buf, sizeof(buf)));
+			json_object_string_add(json_vrf, "interface",
+				ifindex2ifname(bgp->l3vni_svi_ifindex,
+					       bgp->vrf_id));
 			json_object_object_add(json_vrfs, name, json_vrf);
-		} else
+		} else {
 			vty_out(vty,
-				"%4s  %-5d  %-16s  %9u  %10u  %-37s %-10u %-15s\n",
+				"%4s  %-5d  %-16s  %9u  %10u  %-37s\n",
 				type,
 				bgp->vrf_id == VRF_UNKNOWN ? -1
 							   : (int)bgp->vrf_id,
 				inet_ntoa(bgp->router_id), peers_cfg,
-				peers_estb, name, bgp->l3vni,
-				prefix_mac2str(&bgp->rmac, buf, sizeof(buf)));
+				peers_estb, name);
+			if (bgp->inst_type == BGP_INSTANCE_TYPE_VRF)
```

Comment:
Previously L3-VNI and RMAC are printed even for default VRF. Now you are printing this only for non default VRFs. Is there any specific reason for this modification ?

Previous output:
```
 d32ed8711e26# show bgp vrfs
Type  Id     routerId          #PeersVfg  #PeersEstb  Name                                  L3-VNI     Rmac           
 VRF  12     5.1.1.1                   1           1  vrf_11                                0          00:00:00:00:00:00
 VRF  11     5.1.1.1                   1           1  vrf_10                                10         82:7e:46:ca:75:64
 VRF  14     5.1.1.1                   1           1  vrf_13                                13         6e:f0:40:0a:f7:01
 VRF  13     5.1.1.1                   1           1  vrf_12                                12         fe:f4:6f:2f:a6:3a
 VRF  16     5.1.1.1                   1           1  vrf_15                                15         3a:bd:a7:c0:55:01
 VRF  15     5.1.1.1                   1           1  vrf_14                                14         5e:16:46:fe:7b:f4
 VRF  2      5.1.1.1                   1           1  vrf_1                                 1          16:fa:ae:bb:a9:8b
DFLT  0      5.1.1.1                   1           1  default                               11         d6:48:a0:57:78:72
 VRF  4      5.1.1.1                   1           1  vrf_3                                 3          3a:76:ce:1a:0d:8b
 VRF  3      5.1.1.1                   1           1  vrf_2                                 2          be:e0:9a:61:db:d4

```

Reply:
Typically L3VNI and RMAC are configured for non-default vrfs. I have removed the filter.